File: Excel Sheet | Part - 1 - GFG/excel-sheet-part-1.cpp
// { Driver Code Starts
//Initial template for C++


#include<bits/stdc++.h>
using namespace std;

 // } Driver Code Ends
//User function template for C++

class Solution{
    public:
    string ExcelColumn(int N)
   {
       // Your code goes here
       char a[26];
       for(int i=0;i<26;i++){
           a[i]=char(65+i);
       }
       int t=N,y;
       string res="";
       while(t>0){
           y=(t-1)%26;
           res=a[y]+res;
           t=(t-1)/26;
       }
       return res;
   }
};

// { Driver Code Starts.
int main()
{
    int t;
    cin>>t;
    while(t--)
    {
        int n;
        cin>>n;
        Solution ob;
        cout<<ob.ExcelColumn(n)<<endl;
    }
    return 0;
}
      // } Driver Code Ends

File: Excel Sheet | Part - 1 - GFG/excel-sheet-part-1.py
#User function Template for python3

class Solution:
    def ExcelColumn(self, N):
        s=""
        char=[chr(i+ord('A')) for i in range(26)]
        while N>0:
            N,res=divmod(N-1,26)
            s=char[res]+s
        return s
#{ 
#  Driver Code Starts
#Initial Template for Python 3

if __name__ == '__main__':
    t=int(input())
    for tcs in range(t):
        n=int(input())
        ob=Solution()
        print(ob.ExcelColumn(n))

# } Driver Code Ends